setup.py

```diff
@@ -135,20 +135,21 @@ def build_extension(self, ext):
     license="BSD",
     install_requires=[
         "arrow",
-        "numpy",
+        "numpy==1.21.4",
```

Comment:
otherwise, it will pick up `1.22.0rc1` and doesn't find Python 3.7 distribution.

@ejguan  have you seen similar issues before?